components/widgets/elasticsearch/hit-count.js

```diff
@@ -0,0 +1,92 @@
+import { Component } from 'react'
+import fetch from 'isomorphic-unfetch'
+import Widget from '../../widget'
+import Counter from '../../counter'
+
+export default class ElasticsearchErrorHitCount extends Component {
+  static defaultProps = {
+    interval: 1000 * 60 * 5,
+    title: 'Elasticsearch Hit Count'
+  }
+
+  state = {
+    count: 0,
+    error: false,
+    loading: true
+  }
+
+  componentDidMount () {
+    this.fetchInformation()
+  }
+
+  componentWillUnmount () {
+    clearInterval(this.interval)
+  }
+
+  async fetchInformation () {
+    const { url, index, query, auth } = this.props
+
+    try {
+      const res = await fetch(`${url}/${index}/_search?q=${query}`, {
+        credentials: 'included',
+        headers: {
+          // 'Authorization': 'Basic '+btoa('elastic:changeme')
+          'Authorization': `${auth}`
```

Comment:
You can use the variable direct instead of using template literals around the variable 👍🏻
But you could use sting literals for your auth documentation 👌